scripts/create_backup_pd

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+ZONE=us-central1-f
+BUCKET=jenkins-backup-image
+SHA=""
+TOOL_BUCKET=jenkins-backup-image
+
+while getopts z:b:s:t: arg; do
+  case ${arg} in
+    z) ZONE="${OPTARG}";;
+    b) BUCKET="${OPTARG}";;
+    s) SHA="${OPTARG}";;
+    t) TOOL_BUCKET="${OPTARG}";;
+    *) echo "Unrecognized argument ${OPTARG}"; exit 1;
+  esac
+done
+
+if ["$SHA" = ""];then
+  echo 'Missing SHA for secrets. Please use -s'
+  exit 1;
+fi
+
+sudo rm -rf /tmp/jenkins-backup-pd
+sudo mkdir -p /tmp/jenkins-backup-pd/mnt
+
+cd /tmp/jenkins-backup-pd
+sudo dd if=/dev/zero of=disk.raw bs=1024k seek=10240 count=0
+echo '>>>>>>>>>>>>>>>>>>>> Created a blank image.'
+
+sudo losetup -f disk.raw
+sudo losetup -a
+sudo mkfs.ext4 -F -E lazy_itable_init=0,lazy_journal_init=0 /dev/loop0
```

Comment:
Try to find a free loop device, if so, connect to that device and if there is no free loop device available, exit with an error.